lci_strategic_plugin/src/lci_strategic_plugin.cpp

```diff
@@ -362,7 +362,7 @@ void LCIStrategicPlugin::handleGreenSignalScenario(const cav_srvs::PlanManeuvers
 
     resp.new_plan.maneuvers.push_back(composeIntersectionTransitMessage(
         traffic_light_down_track, intersection_end_downtrack_.get(), intersection_speed_.get(),
-        intersection_speed_.get(), light_arrival_time_by_algo, intersection_exit_time, entry_lanelet.id(), exit_lanelet.id()));
+        intersection_speed_.get(), light_arrival_time_by_algo, intersection_exit_time, crossed_lanelets.back().id(), crossed_lanelets.back().id()));
```

Comment:
Same concern as above about these lanelet Ids. Needs a strong justification for this it seems like it would make the maneuver content invalid.

Reply:
crossed_lanelets's last element is the intersection lanelet due to inclusive bound. And this lanelet should be added to start_lanelet_id so that intersectionTransitPlugin correctly passes it to the basic_autonomy library